app-games/src/main/java/com/aptoide/android/aptoidegames/feature_apps/presentation/MyGamesBundleView.kt

```diff
@@ -92,8 +92,9 @@ fun MyGamesBundleViewContent(
       BundleHeader(
```

Comment:
"See All" should also be white.

Reply:
I tested dark and light theme and i belive that the see all text not being in white is a bug in preview

Reply:
tested and confirmed